cedar-policy-cli/src/lib.rs

```diff
@@ -506,6 +513,104 @@ pub fn format_policies(args: &FormatArgs) -> CedarExitCode {
     }
 }
 
+fn generate_schema(path: &Path) -> Result<()> {
+    std::fs::write(
+        path,
+        serde_json::to_string_pretty(&serde_json::json!(
+        {
+            "": {
```

Comment:
Might be good to use a nonempty namespace here since the `"": {` might look strange to first time users.

Reply:
Good point. I think adding a namespace will make entity literals more complicated. We can revisit it later.